boards/ite/it82xx2_evb/Kconfig.it82xx2_evb

```diff
@@ -2,4 +2,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_IT82XX2_EVB
-	select SOC_IT82202_AX
+	select SOC_IT82202AX
```

Comment:
You need to document this change in the release notes and migration guide.

Reply:
Done.